CallLst.pas

```diff
@@ -35,21 +38,25 @@ procedure LoadCallList;
   INDEXBYTES = INDEXSIZE * SizeOf(Integer);
 var
   i: integer;
-  P, Pe: PChar;
-  L: TList;
-
-  FileName: string;
+  P, Pe, myP: PAnsiChar;
+  L: TStringList;
+  mystring: string;
+  myansichar : AnsiChar;
+  mychar : char;
+  mypstring : ^string;
+
+  FileName: AnsiString;

```

Comment:
File name should be Unicode. Use the TFileName type, not AnsiString, it automatically takes the correct string type.